python/tests/avro/test_decoder.py

```diff
@@ -106,9 +107,11 @@ def read(self, size: int = 0) -> bytes:
         self.pos += 1
         return int.to_bytes(1, self.pos, byteorder="little")
 
+    @abstractmethod
```

Comment:
After the [update of mypy](https://mypy-lang.blogspot.com/2022/11/mypy-0990-released.html), methods with empty bodies are not allowed for type safety. Either we turn the class into an abstract one (but the test will fail for now because it has to be readjusted) or we disable this new functionality with the specific flag (`--allow-empty-bodies`). What do you think?

Reply:
Ah, thanks for the explanation. I think it is okay to implement them. For the seek we need to update the `self.pos` and for tell we need to return the pos.

Reply:
Ok, fixed as suggested. Thanks! :)